### DISCUSSION TITLE
Plans to fully support dynamic-scopes feature

### DISCUSSION THREAD

**Discussion by tsahabt** [keycloak/keycloak#29327] [Q&A]
We are keen on utilizing the dynamic-scopes feature on production environment. I've noticed significant progress on the dynamic-scopes Epic (https://github.com/keycloak/keycloak/issues/9225).
Can someone provide any insights into the timeline and plans for incorporating dynamic-scopes into the list of 'Supported' features?

**Comment by codespearhead** [comment]
I suggest closing this discussion in favor of #8532 .

As for the question itself, it’s important to emphasize that Keycloak is an open-source project, so if you're interested in that feature, you can help by upvoting the enhancement proposal and contributing resources toward its development.

If you have a Red Hat subscription, reaching out directly to express your interest might also influence its prioritization.

**Reply by tsahabt** [reply]
Thanks for the response. Just seeking a bit more clarity:

1. Is the dynamic-scopes feature still under development? Is the enhancement complete?
2. What is the plan for promoting dynamic-scopes to become an officially 'supported' feature in Keycloak, transitioning from its current 'experimental' status?
